Assignment/solution_student/css/style.css

```diff
@@ -28,22 +28,212 @@ html {
-/* PAGES
-/*******************************************/
 
-/******************************************
-/* SECTIONS
-/*******************************************/
\ No newline at end of file
+h1 {
+    display: inline-block;
+    margin: 0;
+    color: #f0efef ;
+    font-size: 24px;
+    padding-top: 20px;
+    padding-left: 50px;
+    letter-spacing: 1px;
+}
+
+h1, 
+h4 {
+    font-size: 24px;
+    font-weight: 700;
+    letter-spacing: 1px;
+}
+
+h2 {
+	margin: 0;
+	color: #033048;
+    font-size: 20px;
+    font-weight: 600;
+}
+
+h4 {
+    margin: 0;
+    color: #f9e42e ;
+    font-size: 24px;
+    padding-top: 25px;
+    padding-bottom: 40px;
+    letter-spacing: 1px;
+}
+
+h3{
+	font-size: 14px;
+}
+
+h5 {
+    color: #f0efef ;
+    font-size: 9px;
+    font-weight: 600;
+    padding-top: 20px;
+}
+
+nav {
+    float: right;
+    padding-top: 25px;
+    padding-right: 50px;
+    
+}
+
+li {
+    display: inline;
+}
+
+ul {
+   margin: 0;
+   padding: 0;
+   list-style-type: none;
+}
+
+
+p{
+	color: #606161;
+	font-size: 12px;
+}
+
+main {
+	margin-bottom: 0;
+	padding-top: 25px;
+    padding-left: 50px;
+    padding-bottom: 15px;
+    background-color: #f0efef ;
+    float: left;
+    width: 75%;
+}
+
+a {
+    color: #f9e42e ;
+    text-decoration: none;
+}
+
+footer {
+    height: 240px;
+    margin: 0;
+    background-color: #121212 ;
+    clear: both;
+}
+
+.main-content {
+    float: left;
+    width: 90%;
+}
+
+.main-images{
+	width: 100%;
+	height: auto;
+}
+
+.read-more{
+	color: #033048;
+	text-align: right;
+	font-weight: 700;
+}
+
+.ad-unit{
+	width: 100%;
+	height: auto;
+}
+
+.center-text {
+    text-align: center;
+}
+
+.nav-links {
+	margin: 0;
+    font-size: 14px;
+    font-weight: 300;
+    padding-left: 20px;
+}
+
+
+.sidebar {
+    float: right;
+    width: 25%;
+    margin-bottom: 0;
+    color: #033048;
+    background-color: #f0efef ;
+    padding-right: 15px;
+    padding-left: 0;
+	padding-bottom: 214px;  
+}
+
+.sidebar-li{
+	display: block;
+	font-size: 12px;
+}
+
+.social-media {
+    width: auto;
+    height: 20px;
+    padding-left: 10px;
+    padding-right: 10px;
+    display: inline-block;
+}
+
+.yellow-bar {
+    padding-top: 15px;
+    padding-bottom: 15px;
+    background-color: #f9e42e ;
+}
+
+
+#sign-up-now {
+    color: #f0efef ;
+    font-size: 12px; 
+    font-weight: 600;
+    padding: 6px 40px;
+    border-radius: 3px; 
+    background: #033048 ;
+    text-decoration: none;
+    border: 2px solid #033048 ; /*needed for hover state*/
+}
+
+#relaxr-text {
+	color: #033048;
+	margin-bottom: 0;
+}
+
+#learn-more {
+	color: #033048;
+	margin-top: 0;
+	margin-bottom: 20px;
+	font-weight: 700;
+}
+
+.first-character:first-letter {
+    float: left; 
+    font-size: 60px; 
+    line-height: 60px;  
+    padding-right: 8px; 
+}
+
+a:hover, a#sign-up-now:hover {
+    border: 2px dotted #121212 ;
+}
+
+/*I had alot of trouble getting the grey background to connect 
```

Comment:
Oh! I think I know what you mean, like this: http://d.pr/i/1g2p3 ? Okay! So you're first problem is that you've applied `width: 50%;` to the `body`, so the **entire** page is only going to be 50% width of the browser, so the edges of the containers _can't_ reach the sides ever. What you would want to do is apply the same idea of centering the whole page by setting a width, or max-width, but do it inside each section so that the entire page is actually the full width of the browser, but you're centering the content within each section (header, main area, footer). Sooo, I would remove that `width: 50%` on the body, which will already produce a page that has those backgrounds reaching all the way from the left side of the browser to the right side, but so will all the text and content, and then within each section (inside the `<header>`, between the header and footer, and inside the `<footer>` (but not including the `.yellow-bar` cause you still want that to go edge-to-edge)), add a wrapper div with a common class of something like `.container`, and then in your styles give that container a max-width (for example I just did 1200px because it felt comfortable), and the same `margin: 0 auto;` to keep it centered. If this isn't easy to visualize, I put the modified code together here: https://gist.github.com/poopsplat/6e2bd66177a4370dd35a + https://gist.github.com/poopsplat/6b4dc690126db846c49b